x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/utils/ExceptionsHelper.java

```diff
@@ -19,4 +19,11 @@
         }
         return obj;
     }
+
+    /**
+     * @see org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper#findSearchExceptionRootCause(Throwable)
+     */
+    public static Throwable findSearchExceptionRootCause(Throwable t) {
+        return org.elasticsearch.xpack.core.ml.utils.ExceptionsHelper.findSearchExceptionRootCause(t);
```

Comment:
Is this cross-package dependency ok?
I'd consider introducing `ExceptionsHelper` class under `org.elasticsearch.xpack.core.common` instead.

Reply:
We discussed this with Hendrik. It's not great but harmless otherwise. Adding another `ExceptionsHelper` class is marginally worse. There is much to improve in our exceptions handling framework. I am going to raise an issue that once fixed should make this method unnecessary.